sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/TraceUtil.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.implementation.tracing.TracerProxy;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Signal;
+
+/**
+ * Helper class to help start tracing spans.
+ */
+public class TraceUtil {
+
+    private static final String OPENTELEMETRY_SPAN_KEY = com.azure.core.implementation.tracing.Tracer.OPENTELEMETRY_SPAN_KEY;
+
+    // So this class can't be instantiated.
+    private TraceUtil() {
+    }
+
+    /**
+     * Starts the tracing span for the current service call.
+     *
+     * @param context Context information about the current service call.
+     * @return The updated context containing the span context.
+     */
+    public static Context start(String methodName, Context context) {
+        String spanName = String.format("Azure.eventhubs.%s", methodName);
```

Comment:
`String.format()` is generally slower than just using `+`. You could instead use `"Azure.eventhubs" + methodName`. Given that `start()` call is made every single time a service call is made, we should try and avoid this.

https://kylewbanks.com/blog/java-string-concatenation-vs-stringbuilder-vs-string-format-performance
https://stackoverflow.com/questions/513600/should-i-use-javas-string-format-if-performance-is-important
https://redfin.engineering/java-string-concatenation-which-way-is-best-8f590a7d22a8